lib/connector/home/home_converter.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:logotime/connector/view_model_converter.dart';
+import 'package:logotime/network/model/user/user_model.dart';
+import 'package:logotime/redux/action/user/authorization_action.dart';
+import 'package:logotime/redux/action/user/user_action.dart';
+import 'package:logotime/widget/home/home_view_model.dart';
+
+class HomeConverter extends ViewModelConverter<HomeViewModel> {
+  final UserModel? user;
+  final Function(dynamic) dispatch;
+
+  HomeConverter({required this.user, required this.dispatch});
+
+  @override
+  HomeViewModel build() {
+    return HomeViewModel(
+      (viewModel) => viewModel
+        ..user = user?.toBuilder()
+        ..onLogOutPressed = (() => dispatch(LogOutAction()))
```

Comment:
remove not needed brackets

Reply:
When I remove the brackets the auto-indentation becomes a bit off, like so:
```dart
        ..onLogOutPressed = () => dispatch(LogOutAction())
          ..onUpdatePressed = () => dispatch(GetCurrentUserInfoAction()),
```
If such indentation is okay, I will remove the brackets